### PR TITLE
Update guest-addons

### PR DESCRIPTION
**Pull request by iartemenko**
Add H3ULCB and M3ULCB domain configuration files.

Signed-off-by: Iurii Artemenko <iurii_artemenko@epam.com>

### CODE REVIEW

File: recipes-dom0/dom0-image-thin-initramfs/files/meta-xt-prod-extra/recipes-extended/guest-addons/domu.bb [hunk 1]
```diff
@@ -25,6 +25,8 @@ DOMU_CONFIG_salvator-x-h3-4x2g-xt = "domu-salvator-x-h3-4x2g.cfg"
 DOMU_CONFIG_salvator-xs-h3-xt = "domu-salvator-x-h3.cfg"
 DOMU_CONFIG_salvator-xs-h3-4x2g-xt = "domu-salvator-x-h3-4x2g.cfg"
 DOMU_CONFIG_salvator-xs-h3-2x2g-xt = "domu-salvator-xs-h3-2x2g.cfg"
+DOMU_CONFIG_m3ulcb-xt = "domu-salvator-x-m3.cfg"
+DOMU_CONFIG_h3ulcb-xt = "domu-salvator-x-h3.cfg"
```
Comment (otyshchenko1) [inline recipes-dom0/dom0-image-thin-initramfs/files/meta-xt-prod-extra/recipes-extended/guest-addons/domu.bb]: Please name them appropriate

File: recipes-domd/domd-image-weston/files/meta-xt-prod-extra/recipes-extended/guest-addons/guest-addons.bb [hunk 2]
```diff
@@ -84,6 +84,8 @@ DM_CONFIG_salvator-xs-h3-xt = "dm-salvator-x-h3.cfg"
 DM_CONFIG_salvator-xs-h3-4x2g-xt = "dm-salvator-x-h3.cfg"
 DM_CONFIG_salvator-xs-h3-2x2g-xt = "dm-salvator-x-h3.cfg"
 DM_CONFIG_salvator-x-h3-4x2g-xt = "dm-salvator-x-h3.cfg"
+DM_CONFIG_m3ulcb-xt = "dm-salvator-x-m3.cfg"
+DM_CONFIG_h3ulcb-xt = "dm-salvator-x-h3.cfg"
```
Comment (otyshchenko1) [inline recipes-domd/domd-image-weston/files/meta-xt-prod-extra/recipes-extended/guest-addons/guest-addons.bb]: Please name them appropriate

Comment (iartemenko) [reply recipes-domd/domd-image-weston/files/meta-xt-prod-extra/recipes-extended/guest-addons/guest-addons.bb]: Done

File: recipes-domd/domd-image-weston/files/meta-xt-prod-extra/recipes-kernel/linux/linux-renesas_%.bbappend [hunk 3]
```diff
@@ -132,6 +132,44 @@ KERNEL_DEVICETREE_salvator-x-m3-xt = " \
     renesas/r8a7796-salvator-x-domu.dtb \
 "
 
+##############################################################################
+# M3ULCB
+###############################################################################
+# N.B. DomA device tree is reused from Salvator-X H3 ES3.0 4x2G
+###############################################################################
```
Comment (otyshchenko1) [inline recipes-domd/domd-image-weston/files/meta-xt-prod-extra/recipes-kernel/linux/linux-renesas_%.bbappend]: Please drop this comment. It is unclear at the moment what we should to reuse.

File: recipes-domd/domd-image-weston/files/meta-xt-prod-extra/recipes-kernel/linux/linux-renesas_%.bbappend [hunk 4]
```diff
@@ -132,6 +132,44 @@ KERNEL_DEVICETREE_salvator-x-m3-xt = " \
...
+    renesas/r8a7796-m3ulcb-domd.dtb \
+    renesas/r8a7796-m3ulcb-doma.dtb \
+    renesas/r8a7796-m3ulcb-domu.dtb \
+"
+
+##############################################################################
+# H3ULCB
+###############################################################################
+# N.B. DomA device tree is reused from Salvator-X H3 ES3.0 4x2G
+###############################################################################
```
Comment (otyshchenko1) [inline recipes-domd/domd-image-weston/files/meta-xt-prod-extra/recipes-kernel/linux/linux-renesas_%.bbappend]: Please drop this comment. It is unclear at the moment what we should reuse.

**Comment by otyshchenko1** [comment]
lgtm, but more appropriate name for this PR is "Add skeletons for H3ULCB/M3ULCB" imo

**Pull request closed without merging**